packages/@react-spectrum/list/src/listview.css

```diff
@@ -98,17 +98,43 @@
     padding-bottom: 0px;
   }
 
+  &.is-dragging {
+    border: 1px solid var(--spectrum-global-color-blue-500);
+    border-radius: var(--spectrum-global-dimension-size-50);
+  }
+
   .react-spectrum-ListViewItem-grid {
     display: grid;
-    grid-template-columns: auto auto auto 1fr auto auto;
+    grid-template-columns: auto auto auto auto 1fr auto auto;
     grid-template-rows: 1fr auto;
     grid-template-areas:
-      "checkbox icon image content actions actionmenu chevron"
-      "checkbox icon image description actions actionmenu chevron"
+      "draghandle checkbox icon image content actions actionmenu chevron"
+      "draghandle checkbox icon image description actions actionmenu chevron"
     ;
     align-items: center;
   }
 
+  .react-spectrum-ListViewItem-draghandle-container {
+    grid-area: draghandle;
+    align-items: center;
+    justify-items: center;
+    width: var(--spectrum-global-dimension-size-250);
+  
+    .react-spectrum-ListViewItem-draghandle-button {
```

Comment:
Since the entire item is draggable, and the button only appears on hover, down, and keyboard focus states, I don't think so, but I could be wrong. Not seeing any designs for this state.

Reply:
o, yep, entire item is draggable, i hadn't noticed that